src/compiler/_namespaces/ts.ts

```diff
@@ -2,7 +2,8 @@
 
 export * from "../corePublic";
 export * from "../core";
-export * from "../debug";
+import * as Debug from "../debug";
+export { Debug };

```

Comment:
It turns out that I may actually be able to write:

```ts
export { /** @internal */ Debug }
```

And the comment is preserved and applies, which generally may be a better way to do this in the future.